ethsigner/signer/multiplatform/src/main/java/tech/pegasys/ethsigner/signer/multiplatform/FileBasedSigningMetadataFile.java

```diff
@@ -45,7 +45,7 @@ private String getFilenameWithoutExtension(final String filename) {
     if (filename.endsWith(".toml")) {
```

Comment:
is there a reason we're enforcing a toml extension?

Reply:
If you have the keys and passwords in the same directory, just loading the toml files makes more sense. I don't know - maybe a Q for the PO